lnwallet/channel.go

```diff
@@ -2790,6 +2790,18 @@ func (lc *LightningChannel) evaluateHTLCView(view *HtlcView, ourBalance,
 		NextHeight: nextHeight,
 	}
 
+	// The fee rate of our view is always the last UpdateFee message from
+	// the channel's OpeningParty.
+	openerUpdates := view.Updates.GetForParty(lc.channelState.Initiator())
+	feeUpdates := fn.Filter(func(u *paymentDescriptor) bool {
+		return u.EntryType == FeeUpdate
+	}, openerUpdates)
+	if len(feeUpdates) > 0 {
+		newView.FeePerKw = chainfee.SatPerKWeight(
```

Comment:
Operationally this may be a change but fundamentally it is not. Irrespective of whether the update is locked in or not the feerate with respect to the view is *always* the last fee update issued by the channel initiator.

Reply:
> Irrespective of whether the update is locked in or not the feerate with respect to the view is always the last fee update issued by the channel initiator.

👍 this was what I thought too, glad to have somebody else confirm.